backend/app/enums.py

```diff
@@ -17,3 +17,8 @@ class TransactionCategory(Enum):
     TRANSPORT = "transport"  # ['lime', 'uber'],
     TRAVEL = "reise"  # ['hotel', 'getyourguide', 'tui'],
     OTHERS = "andere"
+
+
+class TransactionType(Enum):
```

Comment:
We already have `TransactionType`. Possible naming would be `Direction` or `Flow`

Reply:
Done